rpm/readline.spec

```diff
@@ -40,16 +61,13 @@ Examples, man and info pages for %{name}.
 %prep
 %autosetup -p1 -n %{name}-%{version}/upstream
 
-pushd examples
-rm -f rlfe/configure
-iconv -f iso8859-1 -t utf8 -o rl-fgets.c{_,}
-touch -r rl-fgets.c{,_}
-mv -f rl-fgets.c{_,}
-popd
-
 %build
-export CPPFLAGS="-I%{_includedir}/ncurses"
-%configure --enable-static=no
+%configure --enable-static=no \
+           --enable-shared			\
+           --enable-multibyte		\
+           --disable-bracketed-paste-default \
+           %{nil}
```

Comment:
Quite a minor detail, but commonly we just skip the last \ when ending the list?

Reply:
Using %{nil} is more convinient when removing on adding or removing lines on commands that spawn over longer lines.